ext/pdo_sqlite/config.w32

```diff
@@ -3,23 +3,15 @@
 ARG_WITH("pdo-sqlite", "for pdo_sqlite support", "no");
 
 if (PHP_PDO_SQLITE != "no") {
-	EXTENSION("pdo_sqlite", "pdo_sqlite.c sqlite_driver.c sqlite_statement.c", null, "/DSQLITE_THREADSAFE=" + (PHP_ZTS == "yes" ? "1" : "0") + " /D SQLITE_ENABLE_FTS3=1 /D SQLITE_ENABLE_FTS4=1 /D SQLITE_ENABLE_FTS5=1 /D SQLITE_ENABLE_JSON1=1 /D SQLITE_ENABLE_COLUMN_METADATA=1 /D SQLITE_CORE=1 /I" + configure_module_dirname + "/../sqlite3/libsqlite /I" + configure_module_dirname);
-	
-	ADD_EXTENSION_DEP('pdo_sqlite', 'pdo');
-	// If pdo_sqlite is static, and sqlite3 is also static, then we don't add a second copy of the sqlite3 libs
-	if (PHP_PDO_SQLITE_SHARED || PHP_SQLITE3_SHARED || PHP_SQLITE3 == 'no') {
-		ADD_SOURCES(configure_module_dirname + "/../sqlite3/libsqlite", "sqlite3.c", "pdo_sqlite");
-	}
-}
+	if (CHECK_LIB("libsqlite3_a.lib;libsqlite3.lib", "pdo_sqlite", PHP_PDO_SQLITE) &&
+		CHECK_HEADER_ADD_INCLUDE("sqlite3.h", "CFLAGS_PDO_SQLITE") &&
+		CHECK_HEADER_ADD_INCLUDE("sqlite3ext.h", "CFLAGS_PDO_SQLITE")
+	) {
+		EXTENSION("pdo_sqlite", "pdo_sqlite.c sqlite_driver.c sqlite_statement.c");
```

Comment:
Just making a note here, are we sure that we don't need the old CFLAGS like SQLITE_ENABLE_FTS3, SQLITE_ENABLE_FTS4, SQLITE_ENABLE_FTS5, SQLITE_ENABLE_JSON1 & SQLITE_CORE1 here?

Reply:
I've grepped for these, and they are not used anywhere. However, if they were, that wouldn't work anyway, since the external libsqlite might have been compiled differently.

Reply:
Noted, then we should be good to go then as I don't see any other obvious things at a glance

Reply:
Yep, those are library flags, not extension flags. Pushed a repo now https://github.com/winlibs/sqlite3.